packages/worker/src/index.ts

```diff
@@ -11,7 +11,7 @@
 export { State, Worker, DataConverter, defaultDataConverter, errors } from './worker';
 export { WorkerOptions, CompiledWorkerOptions } from './worker-options';
 export { ServerOptions, TLSConfig } from './server-options';
-export { Core, CoreOptions } from './core';
+export { Core, CoreOptions, ReplayCore } from './core';
```

Comment:
I'm not sure we want to export it here. Users typically shouldn't use this API

Reply:
It's needed if you want to, for some reason, control your own replay core instance and register workers against it directly. I think for now we can do the lazy-initted core and just use that unless we later discover a reason not to

Reply:
Oh, no there is a good reason, so you can control logging options when desired.